data.sql

```diff
@@ -12,4 +12,50 @@ VALUES ('Charmander', '2020-02-08', 0, false, -11),
   ('Squirtle', '1993-04-02', 3, false, -12.13),
   ('Angemon', '2005-06-12', 1, true, -45),
   ('Boarmon', '2005-06-07', 7, true, 20.4),
-  ('Blossom', '1998-10-13', 3, true, 17);
\ No newline at end of file
+  ('Blossom', '1998-10-13', 3, true, 17);
+
+
+INSERT INTO owners (full_name, age)
+VALUES ('Sam Smith', 34),
+  ('Jennifer Orwell', 19),
+  ('Bob', 45),
+  ('Melody Pond', 77),
+  ('Dean Winchester', 14),
+  ('Jodie Whittaker', 38);
```

Comment:
- [ ] Let us modify our insertion in this way

 INSERT INTO owners (full_name, age)
 VALUES ('Sam Smith', 34);

 INSERT INTO owners (full_name, age)
 VALUES ('Jennifer Orwell', 19);

 INSERT INTO owners (full_name, age)
 VALUES ('Bob', 45);

 INSERT INTO owners (full_name, age)
 VALUES ('Melody Pond', 77);

 INSERT INTO owners (full_name, age)
 VALUES ('Dean Winchester', 14);

 INSERT INTO owners (full_name, age)
 VALUES ('Jodie Whittaker', 38);